app/views/admin/_menu.html.erb

```diff
@@ -158,8 +158,8 @@
         <span class="icon-settings"></span>
         <strong><%= t("admin.menu.title_site_customization") %></strong>
       </a>
-      <ul <%= "class=is-active" if menu_profiles? %>>
-        <li <%= "class=active" if controller_name == "pages" %>>
+        <ul <%= "class=is-active" if menu_customization? %>>
+            <li <%= "class=active" if controller_name == "pages" %>>
```

Comment:
I don't know if these blank spaces before (L161-162) are necessary for a correct indenting ☺️